loopdb/loop.go

```diff
@@ -20,11 +20,11 @@ type SwapContract struct {
 
 	// SenderKey is the key of the sender that will be used in the on-chain
 	// HTLC.
-	SenderKey [33]byte
+	SenderKey []byte
```

Comment:
move this commit to before we add `HtlcScriptV3` + don't intermingle refactors with logical changes (eg `SuccessSequence` returns 10 now?)